Products/CMFPlone/browser/templates/search.pt

```diff
@@ -87,7 +87,7 @@
             </a>
           </div>
 
-          <div class="nav-item dropdown" id="search-filter">
+          <div class="nav-item dropdown me-3" id="search-filter">
```

Comment:
What is this for?

Reply:
adds a minor padding btn `Filter the results` & `Sort by` options...
not neccessary for the functionality at all.. just was looking odd so added it...

![image](https://github.com/user-attachments/assets/efa51179-d99c-49ba-a6a3-994448d580e5)